package/tests/test_api/test_apps.py

```diff
@@ -192,6 +192,19 @@ class TestIndexEndpoint:
     def test_index(self, client):
         response = client.get("/")
         assert response.status_code == 200
+        assert "heap" not in response.text
+
+    @mock.patch("kedro_viz.integrations.kedro.telemetry.get_heap_app_id")
+    @mock.patch("kedro_viz.integrations.kedro.telemetry.get_heap_identity")
+    def test_heap_enabled(
+        self, mock_get_heap_identity, mock_get_heap_app_id, client, tmpdir
+    ):

```

Comment:
We normally do the patching inside the test function, but not sure if there's a different style guide on viz.

Reply:
I guess this is a difference between `unittest.mock` and pytest's mocker?

Reply:
Yea I prefer this style maybe just out of old habits.